addon/authenticators/jwt.js

```diff
@@ -118,14 +109,14 @@ export default TokenAuthenticator.extend({
         // wasn't included in the data object that was passed in.
         const tokenData = this.getTokenData(token);
 
-        expiresAt = this.resolveTime(tokenData[this.tokenExpireName]);
+        expiresAt = tokenData[this.tokenExpireName];
         if (Ember.isEmpty(expiresAt)) {
           return resolve(data);
         }
       }
 
       if (expiresAt > now) {
-        const wait = expiresAt - now - (this.refreshLeeway * 1000);
+        const wait = (expiresAt - now - this.refreshLeeway) * 1000;
```

Comment:
`wait` needs to be in milliseconds for `Ember.run.later`.

Previously `expiresAt` and `now` could be in either seconds or milliseconds which would make the `wait` calculation incorrect if either of them were in seconds.

Now `expiresAt`, `now` and `refreshLeeway` are all in seconds so get added together before being converted to milliseconds.